scripts/server

```diff
@@ -59,21 +59,22 @@ msg "PROGRAM: ${PROGRAM}"
 
 set -e
 
-script_path=$(dirname "$0")
+SCRIPT_PATH=$(dirname "$0")
 # poetry is a mandatory condition to launch this program!
 if [[ -z "${VIRTUAL_ENV}" ]]; then
   error_msg+exit "Error - Launch this script via poetry command:\n\tpoetry run ${PROGRAM}"
 fi
 
 export FLASK_DEBUG=True
 export FLASK_ENV=development
-# Start Worker and Server
-celery worker -A rero_ils.celery --beat -S rero_ils.schedulers.RedisScheduler -l INFO & pid_celery=$!
+# Start Worker and Beat
+celery worker --app rero_ils.celery --loglevel INFO --beat --scheduler rero_ils.schedulers.RedisScheduler & PID_CELERY=$!
 
+# Start Server
 invenio run \
-   --cert "$script_path"/../docker/nginx/test.crt \
-   --key "$script_path"/../docker/nginx/test.key & pid_server=$!
+   --cert "$SCRIPT_PATH"/../docker/nginx/test.crt \
+   --key "$SCRIPT_PATH"/../docker/nginx/test.key & PID_SERVER=$!
 
-trap 'kill $pid_celery $pid_server &>/dev/null' EXIT
+trap 'kill $pid_celery_worker $pid_celery_beat $pid_server &>/dev/null' EXIT
```

Comment:
Where is the `pid_celery_beat`?
Why not in uppercase as other variables?

Reply:
Don't need the `$pid_celery_beat` any more, we can start the worker and beat with one process.